jobs/release-monitoring/branch/integreatly-next/Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+#!groovy
+
+// Checkout remote repository
+def checkoutGitRepo(gitUrl, gitRef, credentialsID) {
+  checkout([$class: 'GitSCM', branches: [[name: gitRef]],
+            doGenerateSubmoduleConfigurations: false,
+            extensions: [],
+            submoduleCfg: [],
+            userRemoteConfigs: [[credentialsId: credentialsID, url: gitUrl]]])
+}
+
+// Get product next branches available in the installation repo
+def getProductNextBranches(gitOrg, gitRepo, sourceBranch) {
+  nextBranches = sh(returnStdout: true, script: "git ls-remote origin | grep next | awk '{print \$2}'").replaceAll("refs/heads/", "").split()
```

Comment:
This is fine for now, but in the future we might to have more control over what branches are actually being merged. This is also going to include the "integreatly-next" branch itself, which i don't think should be an issue, maybe.